### PR TITLE
Loop Religion

### PR DESCRIPTION
**Pull request by Eneocho**
New religion themed around time and the looping cycle, inspired by the constant server restarting accidentally caused by pomf with [this](https://github.com/vgstation-coders/vgstation13/commit/722529493825632bcf98eaee04e755cea00762c5) change.

:cl:
 * rscadd: Time-shifted survivors of the Looping Catastrophe of 2566 have come together to worship and warn of "The Loop", the eternal cycle of restart, where the universe goes back to a previous state. Some might call it deja vu, but they know better.
 * rscadd: Replica time suit, currently only found on Loop chaplains.
 * imageadd: New bible sprite used by Loop bibles. Currently only available to Loop chaplains (not even custom) due to UI code fuckery.

### CODE REVIEW

**Comment by ExpulsionSnooper** [comment]
kino

**Comment by angryturnip** [comment]
Would loop religion chaplains be allowed to reference previous rounds?

**Comment by Eneocho** [comment]
As long as they keep it within IC rules I don't see why not.

**Comment by Kromkar** [comment]
Call it Eternal Return and bake in some references to Rust Cohle in True Detective

**Comment by tinysaturn** [comment]
Eternal Recurrence
we soul sacrifice delta now

**Comment by Hinaichigo** [comment]
> As long as they keep it within IC rules I don't see why not.

IC metagrudging.

**Comment by cloutcode** [comment]
> Call it Eternal Return and bake in some references to Rust Cohle in True Detective

What is that, Nietzsche? Shut the fuck up!

**Comment by DamianX** [comment]
>Currently only available to Loop chaplains (not even custom) due to UI code fuckery.

Can you elaborate on that?

**Comment by Eneocho** [comment]
> > Currently only available to Loop chaplains (not even custom) due to UI code fuckery.
> 
> Can you elaborate on that?

I added the icon but it doesn't appear on the new and fancy chaplain religion selection (it defaults to a regular bible icon on the selection screen) nor the custom religion screen. The icon still works fine when the bible is spawned, but breaks in that screen.
It is also the reason I didn't add more names for the religion's god or bible name, since they looked weird and broken on that screen.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -359,6 +359,17 @@ var/list/tag_suits_list = list()
 	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank/emergency_oxygen,/obj/item/weapon/tank/emergency_nitrogen,/obj/item/toy)
 	body_parts_covered = ARMS|LEGS|FULL_TORSO
 
+/obj/item/clothing/suit/timefake
+	name = "time suit"
+	desc = "A replica of a real time suit, it helps you remember the previous timeline, may the loop protect their souls."
```
Comment (Exxion) [inline code/modules/clothing/suits/miscellaneous.dm]: Can't decide which of the many memes about there being two of something to use but there are two comma splices in here

Comment (Eneocho) [reply code/modules/clothing/suits/miscellaneous.dm]: not a good idea to write your "lore" when tired i tell you that
and the "gee billy" one is a classic you should've immediately chosen

Comment (Exxion) [reply code/modules/clothing/suits/miscellaneous.dm]: Gee Bill was my first thought but I didn't want to either type the whole thing out or risk being called out for half-assing it

**Comment by DamianX** [comment]
I understand what's causing those issues you encountered, I will probably fix them soon

**Comment by WindowsErrors** [comment]
l00p

**Pull request merged**